shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/data/remote/model/SurveyApiModel.kt

```diff
@@ -0,0 +1,37 @@
+package co.nimblehq.ic.kmm.suv.data.remote.model
+
+import co.nimblehq.ic.kmm.suv.data.local.model.SurveyRealmObject
+import co.nimblehq.ic.kmm.suv.domain.model.Survey
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class SurveyApiModel(
+    val id: String,
+    val type: String,
+    val title: String,
+    val description: String,
+    @SerialName("is_active")
+    val isActive: Boolean,
+    @SerialName("cover_image_url")
+    val coverImageUrl: String
```

Comment:
For Android peeps, we always define the `@SerialName` on each property for consistency

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/102/commits/62738814886749f933ffa56e0cd674c12a2b02ad